401-sql-threat-detection-db-policy-multiple-databases/azuredeploy.json

```diff
@@ -0,0 +1,96 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "serverName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name for the SQL server"
+      }
+    },
+    "databaseNames": {
+      "type": "array",
+      "metadata": {
+        "description": "Array of names for the SQL databases"
+      }
+    },
+    "adminUser": {
+      "type": "string",
+      "metadata": {
+        "description": "Username for admin"
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location for server and DBs"
+      }
+    },
+    "adminPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Password for admin"
+      }
+    },
+    "emailAddresses": {
+      "type": "array",
+      "metadata": {
+        "description": "Array of email addresses to receive alerts, each corresponding to its appropriate database (by index in the array)"
+      }
+    }
+  },
+  "variables": {
+    "databaseServerName": "[toLower(parameters('serverName'))]",
+    "databaseNames": "[parameters('databaseNames')]",
+    "databaseServerLocation": "[parameters('location')]",
+    "databaseServerAdminLogin": "[parameters('adminUser')]",
+    "databaseServerAdminLoginPassword": "[parameters('adminPassword')]",
+    "emailAddresses": "[parameters('emailAddresses')]"
+  },
+  "resources": [{
```

Comment:
format the json (e.g. SHIFT+ALT+F in VS Code)

Reply:
You mean formatting in 401-sql-threat-detection-**server-only** template, I presume? That's only one with a small inconsistency (line break between `[` and `{`). All JSONs are formatted, except for that difference in the other template

Reply:
yes, but read the last comment, let's combine these into a single sample first, they are all very similar and it's likely more than one would be applied to a given server/db